integrations/ScriptLoader.js

```diff
@@ -2,17 +2,28 @@
 import logger from "../utils/logUtil";
 
 const defaultAsyncState = true;
+const defaultNonNativeSDKState = "false";
 
 export const LOAD_ORIGIN = "RS_JS_SDK";
 
-const ScriptLoader = (id, src, async = defaultAsyncState) => {
+const ScriptLoader = (
```

Comment:
There are references to the ScriptLoader method in the integrations that need updates (Ex: GoogleOptimize). Check all of them thoroughly.

Reply:
Addressed.